.cirrus.yml

```diff
@@ -1,10 +1,13 @@
+env:
+  CIRRUS_CLONE_DEPTH: 1
+
 container:
-  image: gradle:jdk8
+  image: gradle:5.6.2-jdk8
 
 check_task:
   gradle_cache:
     folder: ~/.gradle/caches
-  check_script: ./gradlew tasks
+  check_script: gradle tasks
```

Comment:
This looks great, but I'm afraid we might forget to update the Gradle container image and then CI builds will be inconsistent from local builds.

Can we get rid of the Gradle container image and instead add `~/.gradle/wrapper/dists/` to the cache?